doc/manuals-ld/installation-guide-centos7.md

```diff
@@ -2,6 +2,20 @@
 
 In order to write this guide, CentOS 7 (Desktop image) was downloaded from [here](http://isoredirect.centos.org/centos/8/isos/x86_64/CentOS-8-x86_64-1905-dvd1.iso), and installed as a virtual machine under VMWare.
 
+## Grant sudo permissions to your user (if you don't have)
+
+```bash
+visudo
+
+# On this line that contains: root ALL=(ALL) ALL, add your user
+<USER> ALL=(ALL) ALL
+```
+
+*Obs.: If you don't have internet access execute this command:*
```

Comment:
That would be valid for each and every command of this guide ... :)

Let's remove it. The users must be able to figure this out by themselves